api/test/h3-data/mocks/create-fake-h3-data.ts

```diff
@@ -1,16 +1,28 @@
 import { getManager } from 'typeorm';
+import { h3Fixtures } from './h3-fixtures';
 
 export const createFakeH3Data = async (
   h3TableName: string,
   h3ColumnName: string,
-): Promise<void> => {
+  bulk?: boolean,
+): Promise<string> => {
   await getManager().query(
     `CREATE TABLE ${h3TableName} (h3index h3index, ${h3ColumnName} float4);` +
       `INSERT INTO ${h3TableName} (h3index, ${h3ColumnName} ) VALUES ('861203a4fffffff', 1000);` +
       `INSERT INTO h3_data ("h3tableName", "h3columnName", "h3resolution") VALUES ('${h3TableName}', '${h3ColumnName}', 6 );`,
   );
+  if (bulk) {
```

Comment:
Fun exercise 1: implement this using Promise.all()
Fun exercise 2: implement this using a single query
Fun exercise 3: explain why this particular implementation sucks

Reply:
Answer for Fun exercise 3: It suck in terms of performance because it makes as much calls as keys in the fixture object are. It was the fastest and easiest way to do it, and since its a test, I thought this could be valid.
Best solution is to make it work in a single query / transaction since its perform a single call and DB takes care of performance in its own realm

Promise.all implementation is not actually THAT essential here since this uses a for of loop, and it actually awaits for async calls, which is not happening in map or forEach loops 

I'll implement this in a single query, cheers